.github/workflows/ci_testing.yml

```diff
@@ -57,3 +57,36 @@ jobs:
           fail_ci_if_error: true
           verbose: true
         if: matrix.os == 'ubuntu-latest' && matrix.python-version == '3.9'
+
+  Build-and-Test-Win:
+    runs-on: windows-2019
+    strategy:
+      matrix:
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - uses: conda-incubator/setup-miniconda@v2
```

Comment:
This needs to have a hash, not a version.

Reply:
Is there a reason we need the hash instead of the version? The build failures were due to an update to scipy

Reply:
Yes - see the DGEC Github actions policy. Everything needs a SHA for security. On all repos.

Reply:
Can I get the hashes from the USGSCSM repo?

Reply:
Yes. They are also discoverable by opening the tag on the action repo (e.g., https://github.com/actions/checkout/tree/v3.5.3) and then clicking the truncated commit hash (screen shot). Then copy the hash from the URL in the newly opened page.

This is required because the action at a given hash is trackable from a security perspective. A tag is mutable, a hash is not.
<img width="914" alt="Screen Shot 2023-07-05 at 2 22 46 PM" src="https://github.com/DOI-USGS/ale/assets/2105468/b7e4cead-b623-424f-9d84-e3ad0c2d0e37">